speculation-rules-security-privacy-questionnaire.md

```diff
@@ -0,0 +1,105 @@
+# Security & Privacy Questionnaire (Speculation Rules)
+
+Covers the [speculation rules explainer](triggers.md). Based on the [W3C TAG Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/).
+
+### What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+At its core the feature does not expose new information to web sites or other parties. However, the user agent may expose information as a consequence of use of this feature.
+
+The feature allows the user agent to use its discretion about which outgoing hypertext references should be prefetched. While we anticipate that user agents will take action to [mitigate user tracking risks](browsing-context.md#privacy-based-restrictions) for cross-origin prefetches, including by denying access to cookies and storage, it remains true that the origin server that serves a prefetch may learn information such as:
+* the request URL
+* the referrer origin from which the prefetch was initiated
+* any information remaining in the request headers which may identify the user agent
+* the client IP address, if it is not anonymized
+
+This is necessary in order to issue an HTTP request to fetch the resource(s).
```

Comment:
"Note that all other fetches on the web platform, e.g. using the `fetch()` API, expose the same information" maybe? You say that below in the general sense, but IMO these specific bullet points can be a bit scary until you actually take a moment to think about them.

Reply:
Done.